lib/services/ios-project-service.ts

```diff
@@ -248,6 +258,13 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 				basicArgs.push("-xcconfig", path.join(projectRoot, this.$projectData.projectName, "build.xcconfig"));
 			}
 
+			// if (this.$logger.getLevel() === "INFO") {
```

Comment:
It partially addresses the https://github.com/NativeScript/nativescript-cli/issues/2006
I've tested it locally but QAs won't have time to push it for this release and I didn't bother to go the right way and make a separate branch with this change only to merge in separate PR later. I hope we will uncomment this shortly after the release.